source/content.js

```diff
@@ -24,6 +24,43 @@ function hideListAddActivity() {
 	$('#stream-items-id .js-activity-list_member_added').css('display', 'none');
 }
 
+function inlineInstagramPosts() {
+	$('a.twitter-timeline-link[data-expanded-url*="//www.instagram.com').each((idx, instagramAnchor) => {
+		const tweetElement = $(instagramAnchor).parents('.js-tweet-text-container');
+		// Don't do anything else if the image is already embedded
+		if ($(tweetElement).siblings('.AdaptiveMedia').length > 0) {
```

Comment:
You don't have to rewrap it in jQuery, it's already a jQuery object: `$(tweetElement)` => `tweetElement`